app/views/tweets/new.html.erb

```diff
@@ -1,3 +1,10 @@
+<% if @tweet.errors.full_messages.any? %>
+  <h4>Errors</h4>
+  <% @tweet.errors.full_messages.each do |error_message| %>
+    <%= error_message if @tweet.errors.full_messages.first == error_message %> <br />
```

Comment:
I have the same concern here as I did about the conditional statement in line four of app/views/tweets/edit.html.erb.  Why only display the first error message?